File: practices/financial_calculator.py
#SA
income=float(input("What is your monthly income?" ))

rent=float(input("What is your monthly rent?" ))

utilities=float(input("What is your monthly utilities? "))

groceries=float(input("What is your monthly groceries? "))

transportation=float(input("What is your monthly transportation?" ))

save=.1*income

excess=income-transportation-utilities-groceries-rent-save

print(f"Your rent is {rent} and that is {round (rent/income*100)}% of your income")
print(f"Your utilities are {utilities} and that is {round (utilities/income*100)}% of your income")
print(f"Your groceries are {groceries} and that is {round (groceries/income*100)}% of your income")
print(f"Your transportation is {transportation} and that is {round(transportation/income*100)} of your income")
print(f"You should save {save} a month, that is {round (transportation/income*100)} of your income")
print(f"You have {excess} of spending money each month")

File: C/notes/practices/financial_calculator.c
//SA financial calculator
#include <stdio.h>

int main () {
    int income;

    int rent;

    int utilities;

    int groceries;

    int transportation;
    
    printf("What is your monthly income? ");
    scanf("%d", &income);
    printf("What is your monthly rent? ");
    scanf("%d", &rent);
    printf("What is your monthly utilities? ");
    scanf("%d", &utilities);
    printf("What is your monthly groceries? ");
    scanf("%d", &groceries);
    printf("What is your monthly transportation? ");
    scanf("%d", &transportation);

}
    float divide(int income, int rent){
        return (float) income/rent;
        }
    
    float divide(int income, int utilities){
        return (float) income/utilities;
       }
     float divide(int income, int groceries){
        return (float) income/groceries;
       }
      float divide(int income, int transportation){
        return (float) income/transportation;
       }

       